### DISCUSSION TITLE
BDI-ZX pcb design?

### DISCUSSION THREAD

**Discussion by tuFFo126** [UzixLS/zx-sizif-512#54] [Q&A]
BDI-ZX ver 2.0 MVcomp FDD interface
I found everything to build a BDI-ZX. The only thing missing is the pcb design. Where can you find the pcb design? Do you have such a project file or access to it? 
This is only jpg.
![BOT](https://github.com/UzixLS/zx-sizif-512/assets/124886433/281e0264-cb05-49bb-a007-cfce35d04a8d)
![TOP](https://github.com/UzixLS/zx-sizif-512/assets/124886433/4b6b0780-91ab-4c3d-8267-28a90bf5c520)

**Comment by UzixLS** [comment]
I'm not event sure if it's gerber files are publicly available or not.
In any case, it would be easy to reproduce.

**Comment by tuFFo126** [comment]
I'm not an electronics expert, does this diagram match this PCB?
[BDI_22.pdf](https://github.com/UzixLS/zx-sizif-512/files/13976796/BDI_22.pdf)

**Reply by UzixLS** [reply]
Yes, looks correct

**Reply by tuFFo126** [reply]
Could you recreate it? Inputs for EPM7064 and EPM7128 are. And for rom zxdiag032.rom.

**Reply by UzixLS** [reply]
If this is offer, than I should say that any payments between RU (where I'm living) and Western countries are difficult for now.

**Reply by tuFFo126** [reply]
I asked as a hobby

**Reply by UzixLS** [reply]
No interest whatsoever. Sorry.

**Reply by UzixLS** [reply]
You may ask MVcomp (who is also known as MV1971 on zx-pk forum) to share his design, may be on PCBWay (they have a monetization model).

**Reply by tuFFo126** [reply]
Thanks, I'll look for it

**Reply by tuFFo126** [reply]
I sent a registration request to the zx-pk forum. Normal registration is disabled. Could you please ask about the BDI-ZX ver.2.0 MVcomp PCB design on my behalf?

**Reply by UzixLS** [reply]
https://t.me/zxpkru_official
